src/edu/ucalgary/ensf409/UI.java

```diff
@@ -63,13 +99,93 @@ public void setQuanity(String quanity) {
 		this.quanity = quanity;
 	}
 	
+	//str to int conversion
+	public int strToInt() {
+		int rst =Integer.parseInt(quanity);
+		return rst;
+	}
+	
+	//Setters have not been implemented for DBURL, username, and password
+	
 	/*processOrder
 	 *Gives information of order details to dataBase SQL class.
-	 *Once information has been processed in back-end, an output of the order
-	 *results will be printed for the user.
 	 */
 	public void processOrder() {
+		Database myOrder = new Database (DBURL, username, password, type, item); //
+		sqlDataStorage = myOrder.getData(); // ex. all mesh chair line items from table stored in array list
+    }
+	
+	/*calculateOrder
+	 *Gives object [] of SQL line item to cheapestCombinations Class.
+	 *Receives new Object [] of cheapest items to fulfill order request.
+	 *Stores in class variable.
+	 */
+	public void calculateOrder() {
```

Comment:
Need to only pass the SQL data object array if calculatecombinations only accepts an Object [] array. Usman and Ritiwik would then need to type cast it within their class, if not and they are expecting arrayLists, then this is fine and I will edit my databases class

Reply:
Sounds good! Apart from what we have discussed, all that is left will be the test class for UI.